addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -70,7 +70,6 @@ public class MongoDbInjectionScanRule extends AbstractAppParamPlugin {
     private static final int BLIND_REQUEST_LIMIT = 5;
     // time-based attack detection will try to take less than the following number of seconds
     // note: detection of this length will generally only happen in the positive (detecting) case.
-    private double BLIND_SECONDS_LIMIT;
```

Comment:
Remove or relocate the comments?

Reply:
removing it because timing utils contains doc string which is sufficient enough to get to know it's working